src/Residue.jl

```diff
@@ -7,11 +7,20 @@
 export name, id, add_atom!, residue_for_atom
 
 """
-Create a new residue with the given ``name`` and an optional residue identifier
+Create a new residue with the given ``name``
 ``resid``.
```

Comment:
This can be removed as well

Reply:
I don't understand, can you please elaborate?

Reply:
sorry, I mean that the `resid` line can be removed as well =)

Reply:
Gotcha